File: Assignments/Assignment_15/p7.c
#include<stdio.h>
void main(){

        int row;
        printf("Enter the no of rows : \n");
        scanf("%d",&row);
        int space=0;
        for(int i=1;i<=row;i++){

                int num=1;
                for(int j=1;j<=row;j++){

                        if(space>=j){

                                printf("\t");
                        }else{

                                printf("%d\t",num);
                                num++;
                        }
                }printf("\n\n");
                space++;
        }
}

File: JAVA/9.Nested_For_2/p7.java
import java.util.*;

class Demo{

	public static void main(String[] args){
	
		Scanner sc = new Scanner(System.in);
		System.out.println("Enter no. of rows: ");
		int row = sc.nextInt();
		for(int i=row;i>0;i--){
		
			for(int j=0;j<row;j++){
			
				System.out.print((i+(j*row))+"\t");
			}System.out.println();
		}
	}
}
